content/docs/concepts/traffic-management/routing/virtual-services/index.md

```diff
@@ -171,6 +180,9 @@ The following diagram shows the configured rule:
     caption="Configurable traffic route based on the namespace of two application services"
     >}}
 
+Note that this approach is also more flexible in that the two destinations could be
```

Comment:
I think I captured the core of what you were saying but I'm not entirely sure.
```suggestion
Using fully qualified hosts in the routing rules provides more flexibility because two destinations in the same namespace can't use the same short name.
```